packages/react/src/contexts/ClerkContextProvider.tsx

```diff
@@ -83,5 +83,11 @@ const useLoadedIsomorphicClerk = (options: IsomorphicClerkOptions) => {
     isomorphicClerk.addOnLoaded(() => setLoaded(true));
   }, []);
 
+  React.useEffect(() => {
+    return () => {
+      IsomorphicClerk.clearInstance();
+    };
+  }, []);
```

Comment:
❓ shouldn't we merge this useEffect with the one above that initializes the ` isomorphicClerk.addOnLoaded` ?

Reply:
The actual initialization happends in the useMemo
`  const isomorphicClerk = React.useMemo(() => IsomorphicClerk.getOrCreateInstance(options), []);`. So i'm not sure, the position shouldn't affect the result tho.